utils/certs/gen_certificates.sh

```diff
@@ -38,146 +38,203 @@
+    # Reset the the CA index
+    rm -f ${CA_HOME}/index.txt ${ph_cert}/serial.txt
+    touch ${CA_HOME}/index.txt
+    echo '01' > ${CA_HOME}/serial.txt
+    echo "Private CA Root Certificate created in ${CA_HOME}"
 }
 
 function generate_agent_cert () {
-	echo "Generating Agent Certificate"
-	# Generate Private key and set its permissions
-	openssl genrsa -out $CERTS/agent.key 4096
-	chmod 0400 $CERTS/agent.key
-	# Generate a Certificate Signing Request (CRS)
-	openssl req -new -config "$CONFIGS/agent.cnf" -key $CERTS/agent.key \
-		-out agent.csr -batch
-	# Create Certificate from request
-	openssl ca -config "$CONFIGS/ca.cnf" -keyfile $PRIVATE/daosCA.key \
-		-cert $CERTS/daosCA.crt -policy signing_policy \
-		-extensions signing_agent -out $CERTS/agent.crt \
-		-outdir $CERTS -in agent.csr -batch
-
-	echo "Required Agent Certificate Files:
-	$CERTS/daosCA.crt
-	$CERTS/agent.key
-	$CERTS/agent.crt"
+    echo "Generating Agent Certificate"
+    # Generate Private key and set its permissions
+    openssl genrsa -out $CERTS/agent.key 4096
+    chmod 0400 $CERTS/agent.key
+    # Generate a Certificate Signing Request (CRS)
+    openssl req -new -config "$CONFIGS/agent.cnf" -key $CERTS/agent.key \
+        -out ${CA_HOME}/agent.csr -batch
+    # Create Certificate from request
+    openssl ca -config "$CA_HOME/ca.cnf" -keyfile $PRIVATE/daosCA.key \
+        -cert $CERTS/daosCA.crt -policy signing_policy \
+        -extensions signing_agent -out $CERTS/agent.crt \
+        -outdir $CERTS -in ${CA_HOME}/agent.csr -batch
+
+    echo "Required Agent Certificate Files:
+    $CERTS/daosCA.crt
+    $CERTS/agent.key
+    $CERTS/agent.crt"
 }
 
 function generate_admin_cert () {
-	echo "Generating Admin Certificate"
-	# Generate Private key and set its permissions
-	openssl genrsa -out $CERTS/admin.key 4096
-	chmod 0400 $CERTS/admin.key
-	# Generate a Certificate Signing Request (CRS)
-	openssl req -new -config "$CONFIGS/admin.cnf" -key $CERTS/admin.key \
-		-out admin.csr -batch
-	# Create Certificate from request
-	openssl ca -config "$CONFIGS/ca.cnf" -keyfile $PRIVATE/daosCA.key \
-		-cert $CERTS/daosCA.crt -policy signing_policy \
-		-extensions signing_admin -out $CERTS/admin.crt \
-		-outdir $CERTS -in admin.csr -batch
-
-	echo "Required Admin Certificate Files:
-	$CERTS/daosCA.crt
-	$CERTS/admin.key
-	$CERTS/admin.crt"
+    echo "Generating Admin Certificate"
+    # Generate Private key and set its permissions
+    openssl genrsa -out $CERTS/admin.key 4096
+    chmod 0400 $CERTS/admin.key
+    # Generate a Certificate Signing Request (CRS)
+    openssl req -new -config "$CONFIGS/admin.cnf" -key $CERTS/admin.key \
+        -out ${CA_HOME}/admin.csr -batch
+    # Create Certificate from request
+    openssl ca -config "$CA_HOME/ca.cnf" -keyfile $PRIVATE/daosCA.key \
+        -cert $CERTS/daosCA.crt -policy signing_policy \
+        -extensions signing_admin -out $CERTS/admin.crt \
+        -outdir $CERTS -in ${CA_HOME}/admin.csr -batch
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]